dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/AbstractBuilder.java

```diff
@@ -17,6 +17,7 @@
 import org.dspace.authorize.service.ResourcePolicyService;
```

Comment:
/**
 * Abstract builder class that holds references to all available services
 *
 * @author Jonas Van Goolen - (jonas@atmire.com)
 * @param <T>
 */